las_geoh5/export_files/driver.py

```diff
@@ -56,7 +56,7 @@ def export_las_files(
 
     drillholes = [k for k in group.children if isinstance(k, Drillhole)]
 
-    print(f"Exporting drillhole surveys and property group data to '{str(basepath)}'")
+    print(f"Exporting drillhole surveys and property group data to '{basepath!s}'")
```

Comment:
**question:** I do not know the "!s" syntax. Is that required? Path usually converts seamlessly to string when used in a string interpolation